packages/suite/src/components/onboarding/SkipStepConfirmation/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, variables } from '@trezor/components';
+import { Translation, Modal } from '@suite-components';
+import { useOnboarding } from '@suite-hooks';
+
+const Wrapper = styled.div`
+    display: flex;
+    width: 100%;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-size: ${variables.NEUE_FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    padding: 16px 0px;
+`;
+
+const Buttons = styled.div`
+    display: flex;
+    width: 100%;
+
+    button + button {
+        margin-left: 24px;
+    }
+`;
+
+interface Props {
+    show: boolean;
+    variant: 'backup' | 'pin';
+    onCancel: () => void;
+}
+
+const getVariant = (variant: Props['variant']) => {
+    if (variant === 'backup') {
+        return {
+            heading: <Translation id="TR_SKIP_BACKUP" />,
+            skipCtaLabel: <Translation id="TR_SKIP_BACKUP" />,
+            children: <Translation id="TR_DO_YOU_REALLY_WANT_TO_SKIP" />,
+        };
+    }
+    return {
+        heading: <Translation id="TR_SKIP_PIN" />,
+        skipCtaLabel: <Translation id="TR_SKIP_PIN" />,
+        children: <Translation id="TR_DO_YOU_REALLY_WANT_TO_SKIP" />,
+    };
+};
+
+const SkipStepConfirmation = ({ variant, show, onCancel }: Props) => {
+    const { goToStep } = useOnboarding();
+    const { children, heading, skipCtaLabel } = getVariant(variant);
+
+    if (!show) return null;
```

Comment:
Wouldn't it make more sense to do this at the call site. Eg.
```jsx
{
    showSkipConfirmation && <SkipStepConfirmation ...>
}
```

and remove this prop from `SkipStepConfirmation` component altogether? From POV of the component: *if you don't need me, then just don't even call me!* .)